src/main/kotlin/no/digdir/catalog_admin_service/service/CodeListService.kt

```diff
@@ -12,4 +14,16 @@ class CodeListService(private val codeListRepository: CodeListRepository) {
 
     fun getCodeListById(catalogId: String, codeListId: String): CodeList? =
         codeListRepository.findCodeListByIdAndCatalogId(codeListId, catalogId)
+
+    fun deleteCodeListById(codeListId: String) =
+        codeListRepository.deleteById(codeListId)
+
+    fun createCodeList(data: CodeListToBeCreated, catalogId: String): CodeList =
+        CodeList(
+            id = UUID.randomUUID().toString(),
```

Comment:
Når jeg tenker meg om, var ikke @jeffreiffers uenig i at vi skulle bruke UUID som id for kodelister

Reply:
Kun hvis ID skal brukes til å kommunisere. Det må vi nesten høre med Breg eller Skatt.

Reply:
Tror det egentlig bare blir for kommunikasjon mellom begrepskatalogen og admin, og at ved eventuell kommunikasjon ut av katalogen hentes verdiene for kodene fra admin før det sendes. Men, ja, vi får ta en prat med brreg og skatt